.circleci/config.yml

```diff
@@ -7,15 +7,34 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - v7-dependency-cache-{{ checksum "yarn.lock" }}
-            - v7-dependency-cache-
+            - v8-dependency-cache-{{ checksum "yarn.lock" }}
+            - v8-dependency-cache-
       - run: yarn
       - run: echo "Checking if lockfiles changed..." && git diff --exit-code
       - run: npm rebuild node-sass
       - save_cache:
-          key: v7-dependency-cache-{{ checksum "yarn.lock" }}
+          key: v8-dependency-cache-{{ checksum "yarn.lock" }}
           paths:
             - node_modules
+            # explicitly list each package node_modules
+            - packages/core/node_modules
+            - packages/datetime/node_modules
+            - packages/docs-app/node_modules
+            - packages/docs-data/node_modules
+            - packages/docs-theme/node_modules
+            - packages/icons/node_modules
+            - packages/karma-build-scripts/node_modules
+            - packages/labs/node_modules
+            - packages/landing-app/node_modules
+            - packages/node-build-scripts/node_modules
+            - packages/select/node_modules
+            - packages/table/node_modules
+            - packages/table-dev-app/node_modules
+            - packages/test-commons/node_modules
+            - packages/test-react15/node_modules
+            - packages/timezone/node_modules
+            - packages/tslint-config/node_modules
+            - packages/webpack-build-scripts/node_modules
```

Comment:
this big blob shaves a full minute off of this step because all the yarn files are where it expects them to be!